client-cli/src/command/wallet_command.rs

```diff
@@ -91,11 +98,12 @@ impl WalletCommand {
             ));
         }
 
-        wallet_client.restore_wallet(name, &passphrase, &mnemonic)?;
+        wallet_client.restore_wallet(name, &enckey, &mnemonic)?;
 
         mnemonic.zeroize();
 
-        success(&format!("Wallet restored with name: {}", name));
+        println!();
+        println!("{}", &hex::encode(enckey.unsecure()));
```

Comment:
Use `success()` instead of `println!()` here. Also add a message (something like: `"Here is your authentication token to be used in future client requests: {}"`)

Reply:
Done, "Authentication token: xxx".